lib/block.mli

```diff
@@ -33,23 +74,36 @@ type t = {
   line : int;
   file : string;
   section : section option;
+  dir : string option;
```

Comment:
I think it can be done in a subsequent PR

Reply:
I'd like to know what the current behaviour is before deciding whether this can be left out for now or not.

I'm happy to take a look myself if you prefer!

Reply:
I've had a quick look and this flag is also used for include blocks (see `update_file_or_block` of bin/test/main)

Reply:
Ok, let's leave it as it is then. Thanks!